std/uuid.d

```diff
@@ -1046,7 +1046,7 @@ unittest
  */
 @trusted UUID randomUUID()
```

Comment:
Hm, in a simple test program, `rndGen()` and its range primitives seem to be safe?
```d
void main() @safe
{
	import std.random : rndGen;
	import std.stdio : writeln;

	auto rnd = rndGen();
	writeln(rnd.empty);
	writeln(rnd.front);
	rnd.popFront();
	auto saved = rnd.save;
}
```
(DMD/Phobos 2.067)